src/services/namespace_service.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import { SuccessCallback } from "../custom_types/custom_types";
+
+class NamespaceService {
+  getNamespaces(successCallback: SuccessCallback) {
+    axios
+      .get("/api/v1/namespaces?timestamp=" + new Date().getTime().toString())
```

Comment:
What's the reason for the timestamp?  Is that to avoid caching or something?

Reply:
Adding a timestamp prevents the browser from grabbing an api call that is in the cache, so it forces the browser to get new data from the api. It was more so a problem on the requests page when using firefox when it would get get an api call that was stored in cache the headers would get dropped so the table navigation would break since the header has information about the number of items.